### PR TITLE
[16.0][MIG] stock_quantity_history_location: Migration to 16.0

### PR DESCRIPTION
**Pull request by slitheryduke11**
The group attribute is added to the 'include_child_locations' field because the module could not be installed otherwise.

Also, a conditional that is never reached is removed because the model's native method always returns a dictionary, making the conditional unnecessary.

### CODE REVIEW

**Comment by slitheryduke11** [comment]
Hi @luisg123v, @JordiBForgeFlow, @ernestotejeda . I think the migration of this module to 16.0 is ready.

I would appreciate it if you could help me with the review.

Thanks in advance!

**Comment by slitheryduke11** [comment]
Hi @luisg123v, @JordiBForgeFlow, @ernestotejeda . I think the migration of this module to 16.0 is ready.

I would appreciate it if you could help me with the review.

Thanks in advance!

**Review by rousseldenis — APPROVED** [review]
Code and functional review

File: stock_quantity_history_location/i18n/stock_quantity_history_location.pot [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+# 	* stock_quantity_history_location
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 15.0\n"
```
Comment (luisg123v) [inline stock_quantity_history_location/i18n/stock_quantity_history_location.pot:7]: 16.0.

Please regenerate the template file

**Review by luisg123v — CHANGES_REQUESTED** [review]
CC @desdelinux

File: stock_quantity_history_location/i18n/stock_quantity_history_location.pot (hunk 1 above)
Comment (rousseldenis) [reply stock_quantity_history_location/i18n/stock_quantity_history_location.pot:7]: @luisg123v FYI the pot file is regenerated at merge and usually is not required to be changed on PR's

Comment (luisg123v) [reply stock_quantity_history_location/i18n/stock_quantity_history_location.pot:7]: @rousseldenis thanks for pointing it out. I was aware `.po` files were regenerated, but I didn't know about `.pot` files also were, I though `.po` were regenerated from the provided `.pot`.

Then this LGTM.

**Review by luisg123v — APPROVED** [review]

**Comment by luisg123v** [comment]
@rousseldenis,

It seems I can't merge here. Could you help me running the merge command, please?

Regards,

File: stock_quantity_history_location/i18n/stock_quantity_history_location.pot (hunk 1 above)
Comment (rousseldenis) [reply stock_quantity_history_location/i18n/stock_quantity_history_location.pot:7]: The .pot is regenerated at merge (of any kind) as if some code has been modified, pot file should be synced with current code. po files are only modified by weblate modifications (several timea per day).

**Comment by rousseldenis** [comment]
> @rousseldenis,
> 
> It seems I can't merge here. Could you help me running the merge command, please?
> 
> Regards,

Indeed, as you have to be already a maintainer of the module to have merge rights.

**Comment by slitheryduke11** [comment]
Hi, @rousseldenis. I've updated the maintainers list to give it a line-by-line format.

**Comment by rousseldenis** [comment]
> Hi, @rousseldenis. I've updated the maintainers list to give it a line-by-line format.

😅 maybe unnecessary if pre-commit does not warn

**Comment by luisg123v** [comment]
@rousseldenis could you re-run the merge command, please?

**Pull request merged**